engine/app/components/citizens_advice_components/sidebar.html.haml

```diff
@@ -0,0 +1,14 @@
+%aside.cads-sidebar
+  .cads-sidebar__content
+    %p.cads-sidebar__title= title
+    - if section_title.url.present?
+      %h3.cads-sidebar__section-title.cads-sidebar__section-title--small
+        = link_to section_title.title, section_title.url
+    - else
+      %h2.cads-sidebar__section-title
+        = section_title.title
+    %nav{ aria: { label: section_title&.title } }
+      %ul.cads-sidebar__links
+        - sidebar_links.each do |link|
+          %li.cads-sidebar__item= link_to link.title, link.url
+      = content
```

Comment:
I think it's fine for the `content` to be rendered into the same nav (it's what happens in public website), we'll need to make sure that this is explicitly mentioned in the docs though, so people know.

Reply:
I have just pushed a change to add it but I'm not sure if the wording is ok

Reply:
Ah OK - yeah I think we need to emphasise the fact that `content` will be rendered into a `nav` element so they'll need to bear that in mind when using it.